test/clt-tests/sharding/syntax/replication-test.rec

```diff
@@ -0,0 +1,96 @@
+––– block: ../base/init –––
+––– input –––
+export INSTANCE=1
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=3
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=4
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/create-cluster –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/join-cluster –––
```

Comment:
It's not clear which node joins the cluster. Let's rename the block to `join-cluster-on-all-nodes` to make it clear.